packages/lib/sync/services/CloseComService.ts

```diff
@@ -120,7 +122,7 @@ export default class CloseComService extends SyncServiceCore implements ISyncSer
         this.log.debug("sync:closecom:web:team:update", { prevTeam, updatedTeam });
         const leadId = await getCloseComLeadId(this.service, { companyName: prevTeam.name });
         this.log.debug("sync:closecom:web:team:update:leadId", { leadId });
-        this.service.lead.update(leadId, updatedTeam);
+        this.service.lead.update(leadId, { companyName: updatedTeam.name });
```

Comment:
Fixing the sync service for updating a team, not related to Sendgrid but found it not working.